src/stormpath.auth.js

```diff
@@ -120,6 +120,9 @@ angular.module('stormpath.auth',['stormpath.CONFIG'])
         var op = $http($spFormEncoder.formPost({
```

Comment:
This is currently a form request. I suggest that you make it a JSON one instead. Also, that you use `$http.post` instead of the `$http` constructor.